man/common

```diff
@@ -0,0 +1,74 @@
+.It Fl e Ar expression
+Evaluates
+.Ar expression .
+.It Fl r Ar file
+Requires
+.Ar file .
+In other words, the specified file is checked for existence at the start of
+.Nm .
+.It Fl S Ar script
+Executes
+.Ar script .
+.It Fl pa Ar directory
+Adds
+.Ar directory
+to the beginning of the code path. If the specified directory already exists, it will be removed from its old position and put to the beginning.
+.Pp
+See also the function
+.Sy :code.add_patha/1 .
+.It Fl pr Ar file
+Does the same thing as
+.Fl r
+.Pq see above
+but in parallel.
+.It Fl pz Ar directory
+Adds
+.Ar directory
+to the end of the code path. If the specified directory already exists, it won't be neither removed from its old position nor put to the end.
+.Pp
+See also the function
+.Sy :code.add_pathz/1 .
+.It Fl -app Ar application
+Starts
+.Ar application
+and all its dependencies.
+.It Fl -cookie Ar value
+Sets the magic cookie to
+.Ar value .
+If the value of the magic cookie isn't specified via the option
+.Fl -cookie
+when the node starts, it will be taken from the file
+.Pa ~/.erlang.cookie
+.Pq see the section Sy FILES .
+Distributed nodes can interact with each other only when their magic cookies are equal.
+.Pp
+See also the function
+.Sy :erlang.set_cookie/2 .
+.It Fl -detached
+Runs the Erlang runtime system detached from the controlling terminal.
+.It Fl -hidden
+Starts a hidden node.
+.Pp
+Connections between nodes are transitive. For example, if a node A is connected to a node B and the node B is connected to a node C, then the node A is connected to the node C. The option
+.Fl -hidden
+allows creating the node which can be connected to a specific node escaping redundant connections.
+.Pp
+The function
+.Sy :erlang.nodes/0
```

Comment:
As far as I know the Kernel.nodes/0 and Kernel.nodes/1 functions don't exist (there are Kernel.node/0 and Kernel.node/1). For the purpose I have to use Node.list/0 and Node.list/1 which are the wrappers for :erlang.nodes/0 and :erlang.nodes/1 respectively. Correct me if I'm mistaken.